tests/test_transforms/test_inference/test_inverse_transform.py

```diff
@@ -133,10 +136,26 @@ def _test_inverse_transform_train_subset_segments(self, ts, transform, segments)
                 ),
                 "regular_ts",
             ),
+            (
+                EmbeddingSegmentTransform(
```

Comment:
Why for `EmbeddingSegmentTransform` we have only test for `TSTCCEmbeddingModel`?

Reply:
we have both